openshift/ocp_on_openstack.adoc

```diff
@@ -89,6 +96,176 @@ Push the `pass` changes to the git repository.
 git push
 ----
 
+== Post installation steps
+
+=== Install jump server
+
+The OCP cluster is created on a private network that allows communication 
+ between the different hosts of the cluster. To be able to SSH to any of 
+ the hosts of this private network a _jump server_ host is required. 
+ This host must be created inside the network of the OCP cluster.
+ 
+To access the _jump server_ a Public IP is required which can be 
+ accomplished by creating a RHOS _Floating IP_ and associating it with that 
+ host.
+
+The OCP private network is created automatically during the installation 
```

Comment:
"The OCP private network is created automatically during the installation 
 process." --> "The private network is created automatically during the execution of the OCP playbook creating the cluster on RHOS."

Reply:
changed